### DISCUSSION TITLE
Is there a way to get the complete list from `upgrade`?

### DISCUSSION THREAD

**Discussion by end-user** [microsoft/winget-cli#4705] [Q&A]
When I do `winget upgrade`, it gives me a list of programs but then says "and 16 more".  Since I don't want to do --all I need to see what the items are.  How can I get an unabridged version of the list?

**Comment by mdanish-kh** [comment]
> "and 16 more"

Not sure if I've seen that exact wording in the `winget upgrade` output before. Would it be possible for you to share a screenshot to show what you're referring to?

**Reply by end-user** [reply]
Drat - I've reduced the list so it's not showing now.  I'll have to update when I have more out of date.
